hopsworks-common/src/main/java/io/hops/hopsworks/common/featurestore/featuregroup/cached/CachedFeaturegroupController.java

```diff
@@ -570,59 +568,6 @@ private void closeConnection(Connection conn, Users user, Project project) {
     }
   }
 
-  /**
-   * Verify user input specific for creation of on-demand training dataset
-   *
-   * @param cachedFeaturegroupDTO the user input data for creating the feature group
-   * @throws FeaturestoreException
-   */
-  @TransactionAttribute(TransactionAttributeType.NEVER)
-  public void verifyCachedFeaturegroupUserInput(CachedFeaturegroupDTO cachedFeaturegroupDTO, Boolean sync)
-    throws FeaturestoreException {
-
-    Pattern namePattern = FeaturestoreConstants.FEATURESTORE_REGEX;
-
-    if(cachedFeaturegroupDTO.getName().length() > FeaturestoreConstants.CACHED_FEATUREGROUP_NAME_MAX_LENGTH ||
-        !namePattern.matcher(cachedFeaturegroupDTO.getName()).matches()) {
-      throw new FeaturestoreException(RESTCodes.FeaturestoreErrorCode.ILLEGAL_FEATUREGROUP_NAME, Level.FINE,
-          ", the name of a cached feature group should be less than "
-          + FeaturestoreConstants.CACHED_FEATUREGROUP_NAME_MAX_LENGTH + " characters and match " +
-          "the regular expression: " +  FeaturestoreConstants.FEATURESTORE_REGEX);
-    }
-
-    if(!Strings.isNullOrEmpty(cachedFeaturegroupDTO.getDescription()) &&
-        cachedFeaturegroupDTO.getDescription().length() >
-          FeaturestoreConstants.CACHED_FEATUREGROUP_DESCRIPTION_MAX_LENGTH){
-      throw new FeaturestoreException(RESTCodes.FeaturestoreErrorCode.ILLEGAL_FEATUREGROUP_DESCRIPTION, Level.FINE,
-          ", the descritpion of a cached feature group should be less than "
-          + FeaturestoreConstants.CACHED_FEATUREGROUP_DESCRIPTION_MAX_LENGTH + " characters");
-    }
-    //Only need to verify the DDL when creating table from Scratch and not Syncing
-    if (!sync) {
-      if (!cachedFeaturegroupDTO.getFeatures().stream().filter(f -> {
-        return (!namePattern.matcher(f.getName()).matches() || f.getName().length()
-          > FeaturestoreConstants.CACHED_FEATUREGROUP_FEATURE_NAME_MAX_LENGTH);
-      }).collect(Collectors.toList()).isEmpty()) {
-        throw new FeaturestoreException(RESTCodes.FeaturestoreErrorCode.ILLEGAL_FEATURE_NAME, Level.FINE,
-          ", the feature name in a cached feature group should be less than "
-            + FeaturestoreConstants.CACHED_FEATUREGROUP_FEATURE_NAME_MAX_LENGTH + " characters and match " +
-            "the regular expression: " + FeaturestoreConstants.FEATURESTORE_REGEX);
-      }
-  
-      if(!cachedFeaturegroupDTO.getFeatures().stream().filter(f -> {
-        if(Strings.isNullOrEmpty(f.getDescription())){
-          f.setDescription("-");
```

Comment:
mmm. I think I read a comment somewhere that said that Hive would fail if you try to create a query with empty comment. How do you handle this case?

Reply:
solved at the point where the table is created.